src/templates/interface/video/dynamic-reveal.js

```diff
@@ -0,0 +1,34 @@
+import { slotService, slotTweaker } from '@wikia/ad-engine';
+import { DEFAULT_VIDEO_ASPECT_RATIO } from '../../outstream/porvata-template';
+
+function add(video, container, params) {
```

Comment:
Please add a sentence describing what "dynamic-reveal" does. I know there is an example but I still think it'd be to some benefit to have a few words written next to code :)

Reply:
Done